provision/service.go

```diff
@@ -311,9 +311,9 @@ func (ps *provisionService) updateGateway(token string, bs SDK.BootstrapConfig,
 	gw.CfgID = bs.MFThing
 	gw.Type = gateway
 
-	th, err := ps.sdk.Thing(bs.MFThing, token)
-	if err != nil {
-		return errors.Wrap(ErrGatewayUpdate, err)
+	th, sdkerr := ps.sdk.Thing(bs.MFThing, token)
+	if sdkerr != nil {
+		return errors.Wrap(ErrGatewayUpdate, sdkerr)
```

Comment:
No need to rename.

Reply:
In the next line, `err` variable-name is used again for `json.Marshal`
But since err is previously of `SDKError interface`, it can't be used to store marshall error of `error interface`.
That's why I had to rename.
But I'll remove the `sdkerr` var from other places where it's not required anymore